app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health: int = 100,
+            hidden: bool = False) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.if_alive(self)
+
+    @classmethod
+    def if_alive(cls, beast: Animal) -> None:
+        cls.alive += [beast] if beast.health > 0 else []
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden is False:
+            self.hidden = True
+        else:
+            self.hidden = False
+
+
+class Carnivore(Animal):
+    def bite(self, beast: Animal) -> None:
+        if isinstance(beast, Herbivore) and beast.hidden is False:
+            beast.health -= 50
+        Animal.alive.remove(beast) if beast.health <= 0 else []
```

Comment:
why `else []`?

Reply:
hello! if i remove else [] - flake8 take me an error:
<img width="894" alt="Screenshot 2024-05-31 at 08 36 57" src="https://github.com/mate-academy/py-herbivores-and-carnivores/assets/105336389/b24ebd35-fa8a-44cc-bc63-4b6775d3f6b5">
  E999 SyntaxError: expected 'else' after 'if' expression

Reply:
<img width="842" alt="Screenshot 2024-05-31 at 08 42 47" src="https://github.com/mate-academy/py-herbivores-and-carnivores/assets/105336389/6a33797b-2548-4e94-9b54-e15c8522072f">